### PR TITLE
Ap data

### PR DESCRIPTION
**Pull request by djangobotpython**

### CODE REVIEW

File: output/views/screening_analytics.py [hunk 1]
```diff
@@ -187,7 +190,16 @@ def get_dist_attendees_avg_att_avg_sc(geog, id, from_date, to_date, partners, va
     if values_to_fetch is None or 'tot_scr' in values_to_fetch:
         sql_values_to_fetch.add('tot_scr')
 
+
     tot_val = run_query(shared_sql.get_totals(geog, id, from_date, to_date, partners, sql_values_to_fetch))[0];
+    # for ap screening
+    if '64' in partners or partners == []:
+        tot_ap_per = shared_sql.ap_screening_overview(geog, id, from_date, to_date, partners)
```
[the remark this answers is not in the document]
Comment (jediknightpython) [reply output/views/screening_analytics.py]: We can remove the hardcoded string and import it from settings.

[the remark this answers is not in the document]
Comment (jediknightpython) [reply output/views/screening_analytics.py]: import from settings

File: output/database/SQL/shared_sql.py [hunk 2]
```diff
@@ -165,6 +166,81 @@ def overview(geog, id, from_date, to_date, partners, type):
...
+        key = 'screening__village_id'
+        ap_total_per = \
+             AP_Screening.objects.filter(screening__date__range=[from_date, to_date], screening__village_id=id) \
+                .values('screening__village_id').annotate(ap_tot_per=Sum('total_members'))
+
+    return (key, ap_total_per)
+
+
+# AP-Bluefrog specific data
+def ap_screening_overview(geog, id, from_date, to_date, partners):
```
[the remark this answers is not in the document]
Comment (jediknightpython) [reply output/database/SQL/shared_sql.py]: Provide some documentation for this function

File: output/views/overview_analytics.py [hunk 3]
```diff
@@ -64,7 +67,21 @@ def overview_module(request):
 
         i['geog'] =  geog_child
 
-
+    # now add the data ap specific data
```
[the remark this answers is not in the document]
Comment (jediknightpython) [reply output/views/overview_analytics.py]: import partner id from settings or db

**Comment by djangobotpython** [comment]
Starting fresh code review and hence closing this.

**Pull request closed without merging**